lib/rules/declaration-block-no-redundant-longhand-properties/index.js

```diff
@@ -47,6 +47,43 @@ const customResolvers = new Map([
 			return `${zipped} / ${columns}`;
 		},
 	],
+	[
+		'transition',
+		(decls) => {
+			/** @type {(input: string | undefined) => string[]} */
+			const commaSeparated = (input = '') =>
+				input
+					.split(',')
+					.map((s) => s.trim())
+					.filter((s) => s.length > 0);
+			const delays = commaSeparated(decls.get('transition-delay')?.value);
+			const durations = commaSeparated(decls.get('transition-duration')?.value);
+			const timingFunctions = commaSeparated(decls.get('transition-timing-function')?.value);
+			const properties = commaSeparated(decls.get('transition-property')?.value);
+
+			if (!(delays.length && durations.length && timingFunctions.length && properties.length)) {
```

Comment:
I'm unsure of how to write a test here that isn't invalid syntax (one of the properties would have to be empty) - any thoughts?

Reply:
How about specifying an empty declaration value (`transition-delay`) like this?

```css
a { transition-delay: ; transition-duration: 0.1s; transition-timing-function: ease; transition-property: top; }
```

Reply:
Thanks for the suggestion! Hm, this seems like invalid CSS to me (though I could be wrong) - would that still make it a good test case?

Reply:
Surely that CSS seems invalid, but PostCSS can parse it as having an empty value. See the example below:

```js
require("postcss").parse("a { color: ; }")
  .walkDecls(({ prop, value }) => console.log({ prop, value }));
//=> { prop: 'color', value: '' }
```

Adding such a case seems to make sense.

Reply:
Gotcha; I'm still a bit confused how to proceed, since that case wouldn't be fixable (or in other words, what would I put in `fixed`), but in that case it seems to break the test infrastructure. 

Alternatively, I could extract this out as a utility and have more rigorous tests for the custom resolvers themselves - what do you think?

Reply:
How about specifying `unfixable: true` like this?

```js
{
	code: 'a { transition-delay: ; transition-duration: 1s; transition-timing-function: ease; transition-property: top; }',
	unfixable: true,
	message: messages.expected('transition'),
},
```